pkg/cedar/intake/models/easi_biz_case.go

```diff
@@ -39,7 +39,7 @@ type EASIBusinessSolution struct {
 	HostingLocation         string `json:"hostingLocation" jsonschema:"description=Where will this solution be hosted,example=AWS"`
 	HostingType             string `json:"hostingType" jsonschema:"description=What type of hosting will this solution use,example=cloud"`
 	Pros                    string `json:"pros" jsonschema:"description=Pros of this solution,example=Will reduce FTE hours needed"`
-	SecurityIsApproved      string `json:"securityIsApproved" jsonschema:"enum=,enum=false,enum=true,description=Is this solution FedRAMP/FISMA approved,example=True"`
+	SecurityIsApproved      string `json:"securityIsApproved,omitempty" jsonschema:"enum=,enum=false,enum=true,description=Is this solution FedRAMP/FISMA approved,example=True"`
```

Comment:
What I had in mind when creating EASI-1579 was to change the type of these fields to `*bool` as well (using a pointer to indicate nullability in our Go code). If you do that, you'll need to leave the `omitempty` tag on, but you can take out the `enum` tags in the `jsonschema:` part of the annotation. You'll also need to change the translation code to use `null.Bool`'s `.Ptr()` method to get a `*bool`.

Reply:
Updated in https://github.com/CMSgov/easi-app/pull/1553/commits/20aa9583e58c0a8f4495ddb39d59fb80c568250b.